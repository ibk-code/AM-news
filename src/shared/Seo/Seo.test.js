import { shallow } from 'enzyme';
import React from 'react';
import Seo from './index';

describe('Seo Component test', () => {
  it('expect to render Seo component', () => {
    expect(shallow(<Seo page="Home" />)).toMatchSnapshot();
  });
});
