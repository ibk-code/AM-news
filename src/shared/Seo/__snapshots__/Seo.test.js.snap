// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seo Component test expect to render Seo component 1`] = `
<Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Beta News- Home
    </title>
    <link
      href="https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;0,900;1,400;1,600&display=swap"
      rel="stylesheet"
    />
  </HelmetWrapper>
  <SkipToContent
    content="main"
  />
</Fragment>
`;
